test/pushy_messaging_tests.erl

```diff
@@ -168,6 +168,7 @@ parse_bad_message_test_() ->
        end},
       {"parse a message with a body that's too large",
```

Comment:
Let's also add coverage for a body that's smaller than a custom size to make sure it's accepted.